docs/content/preview/develop/learn/sql-nosql.md

```diff
@@ -68,7 +68,7 @@ Here is a table that lists some of the important features that YugabyteDB suppor
 
```

Comment:
Operational characteristics can be defined as the runtime concerns that arise when a database is deployed, run, and managed in production. When running a database in production in a cloud-like architecture, a number of operational characteristics become essential. The following table compares the capabilities of YugabyteDB to SQL and NoSQL databases. As with the preceding section, there are a number of NoSQL databases which are different in their own ways and the table below is meant to give a broad idea.

The following table lists some of the important features that YugabyteDB supports, and the API to use to leverage the feature. Note that typically, multiple databases are deployed to achieve the same functionality.